### ISSUE TITLE
For non-iid secenario?

### ISSUE THREAD

**Issue by CaesarGo** [wenh06/fl-sim#5]
Thanks for the very helpful repo. I would like to know if there any interface for non-iid data setting for each client?

I don't find it in the src code.

Thanks.

**Comment by wenh06** [comment]
non-iid of the clients comes from the [data](https://github.com/wenh06/fl-sim/tree/master/fl_sim/data_processing). Most of the federated datasets are distributed in some (fixed) non-iid manner. Because they are inherited from previous research works that already fixed the distributions (perhaps for the ability to reproduce their results). In a few of the federated datasets, one can control how they are distributed across the clients (in such a non-iid manner that one wants). Ideally one should be able to distribute the data across the clients as one wants but this is not implemented yet.
